_sass/base/_default.scss

```
@@ -33,7 +33,7 @@ a {
 } // a
 
 code, kbd, pre, samp {
-  @include font-family(monospace, 'PlexSans');
+  @include font-family(monospace);
```

Fails to build:
```
/opt/build/repo/_sass/base/_default.scss:36:in `font-family': Mixin font-family is missing argument $font. (Sass::SyntaxError)
```

Should this maybe be:

```suggestion
  @include font-family(monospace, 'SourceCodePro');
```

?